eng/pipelines/build.yml

```diff
@@ -47,3 +47,26 @@ jobs:
           condition: succeeded()
           failOnStderr: false
           displayName: 'Publish to GitHub and coverage report'
+  - job: SmokeTest
+    variables:
+      DotNetCoreSDKVersion: 3.0.100
+    pool:
+      vmImage: windows-2019
+    steps:
+      - task: DotNetCoreInstaller@2
+        displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
+        inputs:
+          version: "$(DotNetCoreSDKVersion)"
+      - task: NodeTool@0
+        displayName: "Install Node 13.x"
+        inputs:
+          versionSpec: '13.x'
+      - script: |
+          npm ci
+        displayName: "Install packages"
+      - pwsh: .\eng\Generate.ps1 -Exclude TestServer,TestProjects,Samples -reset -noDebug
```

Comment:
I'd recommend using forward slashes for all file paths in pipeline definitions, as forward slash works on both Windows and Linux.

Reply:
It seems powershell normalized it just fine.